src/engraving/libmscore/element.h

```diff
@@ -135,9 +135,10 @@ class ElementEditData;
 class EditData
 {
     QList<ElementEditData*> data;
+    MuseScoreView* view_ { 0 };
 
 public:
-    MuseScoreView* view              { 0 };
```

Comment:
Not sure why uncrustify doesn't complain about publicly settable pointer fields!